config/test/manager_range_patch.yaml

```diff
@@ -0,0 +1,5 @@
+[
+{"op": "replace",
+ "path": "/data/RANGE_END",
+ "value": "02:00:00:00:00:0F"}
```

Comment:
Isn't it redundant since it's alrady set to `02:00:00:00:00:0F` at `test/kubemacpool.yaml` ?

Reply:
the change in `test/kubemacpool.yaml` is generated by this patch, using kustomize.
we want this change to only affect the test manifest.

Reply:
I am stoopid, hehe ok .